extensions/functions/README.rst

```diff
@@ -0,0 +1,20 @@
+OpenCensus Python Azure Functions Extension
+===========================================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opencensus-extensions-azure-functions.svg
+   :target: https://pypi.org/project/opencensus-extensions-azure-functions/
+
+Installation
+------------
+
+::
+
+    pip install opencensus-extensions-azure-functions
+
+References
+----------
+
+* `OpenCensus Project <https://opencensus.io/>`_
+* `OpenCensus Python SDK <https://github.com/census-instrumentation/opencensus-python/>`_
```

Comment:
Feel free to add a link here to azure functions main page.

Reply:
Yeah, I will add more information on functions prerequisites (e.g. local dev: core tools >= 3.0.3568) into this section.